### PR TITLE
Added message telling user that pickPosition is not supported (for in…

### LINKED ISSUE #6078: Annotate doesn't work in IE in 3D Tiles Interactivity Sandcastle example
http://localhost:8080/Apps/Sandcastle/index.html?src=3D Tiles Interactivity.html&label=All

The annotation does not appear.  I don't think this requires any WebGL extensions that would not be supported in IE.  It works in the other browsers.

The other right click actions - "Print metadata" and "Zoom to feature" - work in IE.

### PR DESCRIPTION
**Pull request by loshjawrence**
Fixes #6078 
Tells the user that pickPosition is not supported for their browser

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Signed CLA is on file.

@loshjawrence, thanks for the pull request! Maintainers, we have a signed CLA from @loshjawrence, so you can review this at any time.

:warning: I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

**Comment by lilleyse** [comment]
Thanks @loshjawrence - add yourself to CONTRIBUTORS.md.

File: Apps/Sandcastle/gallery/3D Tiles Interactivity.html [hunk 1]
```diff
@@ -60,13 +65,13 @@
 var toolbar = document.getElementById('toolbar');
 Cesium.knockout.applyBindings(viewModel, toolbar);
 
-var scene = viewer.scene;
 var annotations = scene.primitives.add(new Cesium.LabelCollection());
 
+
```
Comment (lilleyse) [inline Apps/Sandcastle/gallery/3D Tiles Interactivity.html]: Remove empty line.

File: Apps/Sandcastle/gallery/3D Tiles Interactivity.html [hunk 2]
```diff
@@ -48,7 +48,12 @@
     terrainProvider: Cesium.createWorldTerrain()
 });
 
-viewer.scene.globe.depthTestAgainstTerrain = true;
+var scene = viewer.scene;
+if(!scene.pickPositionSupported) {
```
Comment (lilleyse) [inline Apps/Sandcastle/gallery/3D Tiles Interactivity.html]: Style: we typically put a space between `if` and `(`.

`if (!scene.pickPositionSupported) {`

File: Apps/Sandcastle/gallery/3D Tiles Interactivity.html [hunk 3]
```diff
@@ -48,7 +48,12 @@
...
+    console.log('This browser does not support the pickPosition.');
```
Comment (lilleyse) [inline Apps/Sandcastle/gallery/3D Tiles Interactivity.html]: Remove `the`.

**Comment by lilleyse** [comment]
Looks like CI passed. Thanks @loshjawrence for your first Cesium PR!

**Comment by pjcozzi** [comment]
Congrats @loshjawrence on your first Cesium pull request!

**Pull request merged**